ui/src/data-services/models/taxalist.ts

```diff
@@ -0,0 +1,64 @@
+import { getFormatedDateString } from 'utils/date/getFormatedDateString/getFormatedDateString'
```

Comment:
Nit: Rename file from `ui/src/data-services/models/taxalist.ts` to `ui/src/data-services/models/taxa-list.ts`